books/scss/navbar.scss

```diff
@@ -0,0 +1,31 @@
+.book-navbar {
```

Comment:
I think it's not a book navbar, but a navbar 😝

Reply:
Names have been improved